scss/layout.scss

```diff
@@ -1,10 +1,10 @@
 body {
-  height: 100%;
+  // Usage of variables yield error on SCSS compiler
+  background: linear-gradient(180deg, #dbf8ff 0%, #fcfbf7 88.74%);
```

Comment:
@duranmla we are using a variable for this here https://github.com/debtcollective/discourse-debtcollective-theme/blob/67312f6c18e1a84fed81979f9e534931024763ec/scss/login.scss#L39, is this still an issue?

Reply:
When I mention variables I mean the colours.
I wouldn't expect the whole rule to yield error _(my bad I will update)_
I didn't think into put the whole `linear-gradient(180deg, #dbf8ff 0%, #fcfbf7 88.74%)` into a variable, I was trying to populate the colours with vars